bindings/java/src/main/java/org/apache/opendal/exception/OpenDALException.java

```diff
@@ -19,23 +19,46 @@
 
 package org.apache.opendal.exception;
 
-public class ODException extends RuntimeException {
+/**
+ * A OpenDALException encapsulates the error of an operation. This exception
+ * type is used to describe an internal error from the native opendal library.
+ */
+public class OpenDALException extends RuntimeException {
```

Comment:
After a second consideration, I believe `OpenDALException` is good for Java idiom. This is also how RocksDB names its `RocksDBException` - we don't need save letters here.